bundles/org.openhab.binding.vdr/src/main/java/org/openhab/binding/vdr/internal/VDRHandler.java

```diff
@@ -96,14 +97,16 @@ public void initialize() {
                 con.openConnection();
                 updateStatus(ThingStatus.ONLINE);
                 updateProperties(con);
-            } catch (SVDRPException se) {
+            } catch (SVDRPConnectionException ce) {
                 // also update power channel when thing is offline when initializing
                 thing.getChannels().stream().map(c -> c.getUID()).filter(this::isLinked).forEach(channelUID -> {
                     if (VDRBindingConstants.CHANNEL_UID_POWER.equals(channelUID.getIdWithoutGroup())) {
                         updateState(channelUID, OnOffType.OFF);
                     }
                 });
-                updateStatus(ThingStatus.OFFLINE);
+                updateStatus(ThingStatus.OFFLINE, ThingStatusDetail.NONE, ce.getMessage());
```

Comment:
This seems rather `ThingStatusDetail.COMMUNICATION_ERROR`?

Reply:
My intention was that this is normal offline state, that is not critical or an error. If the VDR cannot be reached because it is turned off I did not want to show it in Error State but as suggested by you with a message on detail page. 
![grafik](https://user-images.githubusercontent.com/76820695/108335300-13d9a180-71d3-11eb-98d4-2ea4f83ba676.png).
If there is a problem (which is represented by SVDRPParseResponseException) then I set ThingStatus to unknown because of a communication error and I don't really know if it's really online or offline. If this is the case then ThingStatus on thing overview page is also displayed as communication error. 

As a VDR is not running 24x7 for many people it's status will be offline regularly because it cannot be reached. So there's no action for the openhab user to do. If there is a problem in communication (and it's reachable) then currently Communication Error is displayed also in Thing list in openHAB. 

If you say that "Communication Error" is the normal state for things that cannot be reached (also in other bindings) then I will change ThingStatusDetail to communication error (also it feels a little wrong for me).

Reply:
I understand your point. But, indeed, other bindings set the Thing status OFFLINE, when their Thing is not reachable. Whether this is supposed to happen on a regular basis or not, doesn't play any role. In fact it is a communication error if the Thing is not reachable, for whatever reason.

The UNKNOWN status should only be set during initialization. See the definition https://www.openhab.org/docs/concepts/things.html#thing-status

Reply:
Thanks for your explanation, Fabian. I now set the communication error in both cases. And thanks for your patience